tickit/adapters/epicsadapter/adapter.py

```diff
@@ -76,23 +78,13 @@ def load_records_without_DTYP_fields(self):
         softioc.dbLoadDatabase(out.name, substitutions=f"device={self.ioc_name}")
         os.unlink(out.name)
 
-    def build_ioc(self) -> None:
-        """Builds an EPICS python soft IOC for the adapter."""
-        builder.SetDeviceName(self.ioc_name)
-
-        self.load_records_without_DTYP_fields()
-        self.on_db_load()
-
-        softioc.devIocStats(self.ioc_name)
-
-        builder.LoadDatabase()
-        event_loop = asyncio.get_event_loop()
-        dispatcher = asyncio_dispatcher.AsyncioDispatcher(event_loop)
-        softioc.iocInit(dispatcher)
-
     async def run_forever(
         self, device: Device, raise_interrupt: RaiseInterrupt
     ) -> None:
         """Runs the server continously."""
         await super().run_forever(device, raise_interrupt)
-        self.build_ioc()
+        builder.SetDeviceName(self.ioc_name)
+        self.load_records_without_DTYP_fields()
+        self.on_db_load()
```

Comment:
Even though subclasses of `EpicsAdapter` can get `builder` directly, it may be nice to pass it here as if we were to switch to some alternative which didn't utilise a singleton it would be required.

Reply:
I considered this, main problem is that `builder` is a module so if I pass it as a variable I lose auto-completion in my IDE.